build.gradle

```diff
@@ -65,6 +65,7 @@ subprojects {
     targetCompatibility = 1.8
 
     repositories {
+        mavenLocal()
```

Comment:
should this be here? I add for testing but should it be here for the real thing?